[Feature][doc] Refactor and separate the Resource Center Document (#9658)
apache/dolphinscheduler: docs/docs/en/guide/resource/configuration.md

yarn.job.history.status.address=http://localhost:19888/ds/v1/history/mapreduce/jobs/%s

# datasource encryption enable
datasource.encryption.enable=false

# datasource encryption salt
datasource.encryption.salt=!@#$%^&*

# data quality option
data-quality.jar.name=dolphinscheduler-data-quality-dev-SNAPSHOT.jar

#data-quality.error.output.path=/tmp/data-quality-error-data

# Network IP gets priority, default inner outer

# Whether hive SQL is executed in the same session
support.hive.oneSession=false

# use sudo or not, if set true, executing user is tenant user and deploy user needs sudo permissions;
# if set false, executing user is the deploy user and doesn't need sudo permissions
sudo.enable=true

# network interface preferred like eth0, default: empty
#dolphin.scheduler.network.interface.preferred=

# network IP gets priority, default: inner outer
#dolphin.scheduler.network.priority.strategy=default

# system env path
#dolphinscheduler.env.path=env/dolphinscheduler_env.sh

# development state
development.state=false

# rpc port
alert.rpc.port=50052
```

> **_Note:_**
>

> > * If only the `api-server/conf/common.properties` file is configured, then only resource uploading is enabled, not normal use. If you want to execute the files in the workflow you need to configure `worker-server/conf/common.properties` additionally.

Reviewer: ```suggestion
> * If only the `api-server/conf/common.properties` file is configured, then only resource uploading is enabled, you can not use resources in task. If you want to use or execute the files in the workflow you need to configure `worker-server/conf/common.properties` too.
```
